utils/misc.py

```diff
@@ -1,3 +1,104 @@
+import asyncio
+from datetime import datetime, timedelta
+
+from tortoise import timezone
+
+
+DEGENERACY_DELAY = timedelta(seconds=1)  # when datetime is in the past, a small amount of time is slept for regardless
```

Comment:
I put it in to avoid rate-limiting, i.e. when the bot queues all the existing timers at the start, this stops it from sending all those messages at once.